### ISSUE TITLE
Requesting info on expanding supported languages

### ISSUE THREAD

**Issue by Yayi98** [EmilioK0/pydeepl#15]
How can the support for more languages be expanded? How difficult would it be to add a functionality to support Japanese as a source language? I have limited experience in python but I'd like to contribute towards this feature.

**Comment by me-suzy** [comment]
yes, indeed. DeepL should support all the languages, just as google-translate. Especially for Python Scripts. Right now, I am using google-translate Python library, because I have my website translated in 30 languages.

By the way, I try to install with CMD the library of DeepL , and it's not working. How can I test it on Python scritps?

`py- m pip install pydeepl` - **doesn't work on CMD**
